src/m_menu.c

```diff
@@ -6602,6 +6602,8 @@ void M_StartControlPanel (void)
   currentMenu = &MainDef;         // JDC
   itemOn = currentMenu->lastOn;   // JDC
   print_warning_about_changes = false;   // killough 11/98
+
+  G_ClearInput();
```

Comment:
Not sure if `M_StartControlPanel()` and `M_ClearMenus()` are the best places to call this function.

Reply:
I think it's okay. Do we still need https://github.com/fabiangreffrath/woof/pull/1460?

Reply:
I think it's still a good idea.